### PR TITLE
Introducing vendor/sunnyvalley plugin.

### PR DESCRIPTION
**Pull request by muratbalaban43**
This plugin contains the vendor package repository information.
Vendor-provided package repository hosts os-sensei package and related dependencies.

https://github.com/opnsense/plugins/issues/1394

### CODE REVIEW

File: vendor/os-sunnyvalley/Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=sunnyvalley
+PLUGIN_VERSION=1.0
+PLUGIN_DEVEL=
```
Comment (fabianfrz) [inline vendor/os-sunnyvalley/Makefile]: yes or remove

File: vendor/os-sunnyvalley/pkg-descr [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Sensei is a plugin for networking software complementing them with
+state of the art next generation network security features, such as
```
Comment (fabianfrz) [inline vendor/os-sunnyvalley/pkg-descr]: Imho: This plugin adds a proprietary repository to install Sensei, which is...


Just to say more clearly what it really does.

File: vendor/os-sunnyvalley/pkg-descr [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+* Application Control
+* Cloud Application Control
+* Advanced Network Analytics
+* All-ports Full TLS Inspection
+* Cloud Threat Intelligence
+* Web Security & Web Filtering
+* Active Directory Integration
+
+Note: Sensei is a propriatery commercial project.
```
Comment (fabianfrz) [inline vendor/os-sunnyvalley/pkg-descr]: s/project/product

File: vendor/os-sunnyvalley/pkg-descr [hunk 4]
```diff
@@ -0,0 +1,20 @@
+Sensei is a plugin for networking software complementing them with
+state of the art next generation network security features, such as
+
+* Application Control
+* Cloud Application Control
+* Advanced Network Analytics
+* All-ports Full TLS Inspection
+* Cloud Threat Intelligence
+* Web Security & Web Filtering
+* Active Directory Integration
```
Comment (fabianfrz) [inline vendor/os-sunnyvalley/pkg-descr]: Shouldn't that be mainly in the Sensei plugin descripion?

**Comment by AdSchellevis** [comment]
@fabianfrz thanks for reviewing

@muratbalaban43 fixed minor typo in https://github.com/opnsense/plugins/commit/8d19ee120dabde4f9d4a14c5cd8f7f66664b77b8, should  be good now.

**Pull request merged**